CudaArchitectureSelector.cmake

```diff
@@ -230,14 +238,15 @@ function(cas_get_architectures_by_name name output)
     set( kepler_version 3)
     set(maxwell_version 5)
     set( pascal_version 6)
-    set(  volta_version 7)
+    set(  volta_version "7(0|2)")
+    set( turing_version 75)
     set( ampere_version 8)
```

Comment:
You could also probably add 86 here as well ? [Reference: compute capability list](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities)

Reply:
86 is still Ampere so that will be matched all fine by the regex